### ISSUE TITLE
500 Internal Server Error in script generation

### ISSUE THREAD

**Issue by leeping** [openmm/openmm-setup#23]
In a simple use case for openmm-setup, I loaded a PDB file containing a water box and I went with all the default options. The option to clean up the structure was set to "No. My file is all ready to simulate".

In the script window that followed, the following error message was displayed instead of the script (see screenshot):

```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

At the same time, the terminal displayed these error messages:
```
127.0.0.1 - - [11/Dec/2020 10:21:40] "POST /setSimulationOptions HTTP/1.1" 500 -
[2020-12-11 10:21:42,356] ERROR in app: Exception on /setSimulationOptions [POST]
Traceback (most recent call last):
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/openmmsetup/openmmsetup.py", line 305, in setSimulationOptions
    return createScript()
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/openmmsetup/openmmsetup.py", line 469, in createScript
    pdbType = session['pdbType']
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/werkzeug/local.py", line 377, in <lambda>
    __getitem__ = lambda x, i: x._get_current_object()[i]
  File "/Users/leeping/opt/miniconda3/envs/che211a/lib/python3.7/site-packages/flask/sessions.py", line 84, in __getitem__
    return super(SecureCookieSession, self).__getitem__(key)
KeyError: 'pdbType'
```

When I tried to set the options in the script window, it did not result in any changes to the displayed text.

However, if I choose the option "Yes. Let's clean it up now.", the script window works properly with no errors.  

If I close the tab, load the same file, and choose "No", the error message is no longer displayed.  However, I am able to reproduce the error message by repeating the process in a private window, which means that it might be browser cache-related.

I'm also attaching a screenshot and the PDB file that was loaded to reproduce the error.
<img width="1421" alt="Screenshot 2020-12-11 at 10 22 22 AM" src="https://user-images.githubusercontent.com/1441560/101940242-0b456900-3b9b-11eb-9724-267818d70578.png">
[waterbox.pdb.zip](https://github.com/openmm/openmm-setup/files/5680891/waterbox.pdb.zip)

**Comment by peastman** [comment]
I think this is fixed in 1.2 (just released today).  Can you update to the latest version and see if it now works?

```
conda install -c conda-forge openmm-setup
```

**Comment by leeping** [comment]
I just checked that it works.  Thanks a lot!
